packages/webcrypto-ed25519-polyfill/src/secrets.ts

```diff
@@ -131,10 +131,54 @@ export function verifyPolyfill(key: CryptoKey, signature: BufferSource, data: Bu
     if (key.type !== 'public' || !key.usages.includes('verify')) {
         throw new DOMException('Unable to use this key to verify', 'InvalidAccessError');
     }
-    const publicKeyBytes = ed25519.getPublicKey(getSecretKeyBytes_INTERNAL_ONLY_DO_NOT_EXPORT(key));
+    const storedBytes = getSecretKeyBytes_INTERNAL_ONLY_DO_NOT_EXPORT(key);
     try {
-        return ed25519.verify(bufferSourceToUint8Array(signature), bufferSourceToUint8Array(data), publicKeyBytes);
+        return (
+            ed25519.verify(
+                bufferSourceToUint8Array(signature),
+                bufferSourceToUint8Array(data),
+                ed25519.getPublicKey(storedBytes)
+            ) || ed25519.verify(bufferSourceToUint8Array(signature), bufferSourceToUint8Array(data), storedBytes)
+        );
     } catch {
         return false;
     }
 }
+
+export function importKeyPolyfill(
+    format: KeyFormat,
+    keyData: BufferSource,
+    extractable: boolean,
+    keyUsages: readonly KeyUsage[]
+): CryptoKey {
+    const type =
+        // could be more rigorous
+        format === 'pkcs8' ? 'private' : 'public';
```

Comment:
I think a better way to structure this would be to branch the code on `format`, something like this:

```ts
if (format === 'raw') {
  // you might need some checks here that you're not getting from `createKeyPairFromBytes`
  // code to import + return the public key
} else if (format === 'pkcs8') {
  // code to import + return the private key
} else {
  // TODO: coded error
  throw new Error(`cannot import key with format ${format} format`)
}
```